db/seeds.rb

```diff
@@ -8,15 +8,58 @@
 #   movies = Movie.create([{ name: 'Star Wars' },
 # { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+#
+#
+#
+require 'faker'
+
+Answer.destroy_all
+Question.destroy_all
+Test.destroy_all
+Technology.destroy_all
+User.destroy_all
+
+users = [
+  ['fcbb99', '123456', 'fcbb99@mail.ru', '2018-12-31 15:32:10'],
```

Comment:
This format requires you to remember the order of columns (which is absolutely unnecessary). You could make it more readable by organizing it the following way:
``` ruby
[
  {
    username: 'fcbb99',
    password: '123456',
    email: 'fcbb99@mail.ru',
    confirmed_at: '2018-12-31 15:32:10'
  },
  {
    # ...
  },
  # ...
].each do |user_attributes|
  User.create(user_attributes)
end
```

Also please pay more attention to naming. Your `users` variable doesn't hold users (i.e. instances of the `User` class). This variable should've been called `users_data`